service/share/autoyast-compat.json

```diff
@@ -247,8 +247,43 @@
   },
   {
     "key": "services-manager",
-    "support": "planned",
-    "notes": "Meanwhile you can use post-installation scripts to handle these cases."
+    "support": "yes",
```

Comment:
Please, add a note explaining that the section is automatically converted to a post-script.